src/test/emulators/auth/deleteAccount.spec.ts

```diff
@@ -103,4 +105,42 @@ describeAuthEmulator("accounts:delete", ({ authApi }) => {
         expect(res.body.error).to.have.property("message").equals("MISSING_LOCAL_ID");
       });
   });
+
+  it("should error on delete with idToken if usageMode is passthrough", async () => {
+    const { idToken } = await registerUser(authApi(), {
+      email: "alice@example.com",
+      password: "notasecret",
+    });
+    await deleteAccount(authApi(), { idToken });
+    await updateProjectConfig(authApi(), { usageMode: "PASSTHROUGH" });
+
+    await authApi()
+      .post("/identitytoolkit.googleapis.com/v1/accounts:delete")
+      .send({ idToken })
+      .query({ key: "fake-api-key" })
+      .then((res) => {
+        expectStatusCode(400, res);
+        expect(res.body.error)
+          .to.have.property("message")
+          .equals("UNSUPPORTED_PASSTHROUGH_OPERATION");
+      });
+  });
+
+  it("should error on delete with localId if usageMode is passthrough", async () => {
+    const { localId, idToken } = await registerUser(authApi(), {
+      email: "alice@example.com",
+      password: "notasecret",
+    });
+    await deleteAccount(authApi(), { idToken });
```

Comment:
IMO we should just remove creating and then deleting the account, it only confuses. We can just do `localId: "does-not-exist"` below.

Reply:
Sounds good, done!